staging/src/k8s.io/code-generator/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 
 require (
 	github.com/emicklei/go-restful/v3 v3.11.0 // indirect
+	github.com/fxamacker/cbor/v2 v2.7.0 // indirect
```

Comment:
Any ideas where this change comes from?

Reply:
```shell
daman:code-generator/ (bump-k8s-utils) $ pwd                                                                                                                                                   [18:51:00]
/Users/daman/github.com/aroradaman/kubernetes/staging/src/k8s.io/code-generator
daman:code-generator/ (bump-k8s-utils) $ go mod why -m github.com/fxamacker/cbor/v2                                                                                                            [18:52:25]
# github.com/fxamacker/cbor/v2
k8s.io/apimachinery/pkg/runtime/serializer/cbor
github.com/fxamacker/cbor/v2
daman:code-generator/ (bump-k8s-utils) $                                              
```
https://github.com/kubernetes/kubernetes/pull/125669 bumped `cbor` to  `v2.7.0` but the commit This https://github.com/kubernetes/kubernetes/commit/dbe4c093d9f5b85fa509042556edf61fb6503b22 doesn't carry the same change in `staging/src/k8s.io/code-generator/go.mod`